### PR TITLE
Create raspios/buster Branch with Debianization [REVPI-3011]

### PR DESCRIPTION
**Pull request by nbuchwitz**

### CODE REVIEW

**Comment by nbuchwitz** [comment]
NOTE: This needs to be changed to raspios/buster as target branch before merging. This branch does not exist yet (will be forked from master after PR #6 in merged)

**Comment by iluminat23** [comment]
Just push a raspios/buster branch which is the same as the master and then recreate the PR to this branch.

File: debian/gbp.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[DEFAULT]
+upstream-tag = v%(version)s
+debian-branch = raspios/buster
+debian-tag = raspios/%(version)s
+debian-tag-msg = %(pkg)s RaspiOS release %(version)s
+pristine-tar = True
```
Comment (iluminat23) [inline debian/gbp.conf]: Missing \n at EOF

File: debian/revpi-repo.install [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/dh-exec
```
Comment (iluminat23) [inline debian/revpi-repo.install]: don't use dh-exec!

File: debian/rules [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/make -f
+# See debhelper(7) (uncomment to enable)
+# output every command that modifies files on the build system.
+#export DH_VERBOSE = 1
```
Comment (iluminat23) [inline debian/rules]: remove useless comments

**Review by iluminat23 — CHANGES_REQUESTED** [review]
The github webui doesn't show the file permissions. Make sure that they are ok. Some might be +x which is not needed when we don't use dh-exec.

File: debian/revpi-repo.install (hunk 2 above)
Comment (nbuchwitz) [reply debian/revpi-repo.install]: Right. Needs https://github.com/RevolutionPi/revpi-repo/pull/8 before I can change it here.

**Comment by nbuchwitz** [comment]
Bullseye will follow as a rebase after this one is reviewed

**Review by iluminat23 — APPROVED** [review]
LGTM

**Pull request merged**